build

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Whoops, I don't think my comment went through. The style guide requires /bin/bash:

>Executables must start with #!/bin/bash

https://google.github.io/styleguide/shell.xml?showone=Which_Shell_to_Use#Which_Shell_to_Use